UITests/BraintreeDropIn_UITests.swift

```diff
@@ -567,13 +570,13 @@ class BraintreeDropIn_PayPal_UITests: XCTestCase {
             }
             return true
         }
-        app.tap()
+        app.coordinate(withNormalizedOffset: CGVector.zero).tap()
```

Comment:
My theory is that `app.tap()` was usually tapping in the middle of the screen, right on the "Proceed with Sandbox Purchase" button on the PayPal site, which cause the flow to finish before the next lines of the test had a chance to run. If the tap occurs in the corner of the screen, we should be able to avoid this problem.

Reply:
Awesome find!!!